project.mml

```diff
@@ -1681,11 +1681,11 @@ Layer:
               tags->'memorial' as memorial,
               tags->'artwork_type' as artwork_type,
               tags->'vending' as vending,
-              CASE WHEN shop IN ('supermarket', 'bag', 'bakery', 'beauty', 'bed', 'bookmaker', 'books', 'butcher', 'carpet', 'clothes', 'computer',
-                                'confectionery', 'fashion', 'convenience', 'department_store', 'doityourself', 'hardware', 'fabric', 'fishmonger', 'florist',
-                                'garden_centre', 'hairdresser', 'hifi', 'car', 'car_repair', 'bicycle', 'mall', 'pet',
-                                'photo', 'photo_studio', 'photography', 'seafood', 'shoes', 'alcohol', 'gift', 'furniture', 'kiosk',
-                                'mobile_phone', 'motorcycle', 'musical_instrument', 'newsagent', 'optician', 'jewelry', 'jewellery',
+              CASE WHEN shop IN ('supermarket', 'bag', 'bakery', 'beauty', 'bed', 'bookmaker', 'books', 'butcher', 'carpet', 'clothes', 
+                                'computer', 'confectionery', 'fashion', 'convenience', 'department_store', 'doityourself', 'hardware', 
+                                'fabric', 'fishmonger', 'florist', 'garden_centre', 'hairdresser', 'hifi', 'car', 'car_repair', 'bicycle', 
+                                'mall', 'pet', 'photo', 'photo_studio', 'photography', 'seafood', 'shoes', 'alcohol', 'gift', 'furniture', 
+                                'kiosk', 'mobile_phone', 'motorcycle', 'musical_instrument', 'newsagent', 'optician', 'jewelry',
```

Comment:
Can you remove the rewrapping of the other lines so this part of the diff is just one line?

I'd rather have uneven line lengths than more difficulty with `git bisect`

Reply:
Sure, sorry.